src/main/java/io/redlink/more/data/model/garmin/GarminSummaryType.java

```diff
@@ -1,20 +1,30 @@
 package io.redlink.more.data.model.garmin;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public enum GarminSummaryType {
     DAILIES,
     EPOCHS,
-    STRESS,
+    STRESSDETAILS,
     PULSEOX,
-    SLEEP,
+    SLEEPs,
```

Comment:
```suggestion
    SLEEPS,
```
Das sollte doch sicher ein upper-case `S` sein, oder?